services/autoscaling/src/simcore_service_autoscaling/modules/ec2.py

```diff
@@ -37,7 +40,15 @@ def _is_ec2_instance_running(instance: ReservationTypeDef):
     )
 
 
-@dataclass
+@dataclass(frozen=True)
+class EC2InstanceData:
+    launch_time: datetime.datetime
```

Comment:
Maybe add a comment which TZ (UTC?) this here has for future reference

Reply:
the launch_time is filled up by the AWS SDK. therefore they define the time zone